pennylane_lightning/core/lightning_base.py

```diff
@@ -556,3 +551,87 @@ def shape(var):
 
         shapes = [shape(var) for var in jaxpr.outvars]
         return jax.pure_callback(evaluator, shapes, consts, *args, vectorized=False)
+
+    def jaxpr_jvp(
+        self,
+        jaxpr: "jax.core.Jaxpr",
+        args: Sequence[TensorLike],
+        tangents: Sequence[TensorLike],
+        execution_config: Optional[ExecutionConfig] = None,
+    ) -> tuple[Sequence[TensorLike], Sequence[TensorLike]]:
+        """
```

Comment:
I think the docstring should mention that only the adjoint method is implemented at the moment :)